### PR TITLE
New package: openimagedenoise-1.1.0

### PR DESCRIPTION
**Pull request by teldra**

### CODE REVIEW

**Comment by xtraeme** [comment]
Looks like it's x86 specific and does not support i686. You can set `archs="x86_64*"`

File: srcpkgs/openimagedenoise/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+distfiles="https://github.com/OpenImageDenoise/oidn/releases/download/v${version}/oidn-${version}.src.tar.gz"
+checksum=4dd484abea8a0b3d12d346343fcb1ab7abef8f94318d8c537f69a20c2a75c4eb
+
+openimagedenoise-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/*.so
+	}
```
Comment (xtraeme) [inline srcpkgs/openimagedenoise/template]: missing `vmove usr/lib/cmake`

Comment (teldra) [reply srcpkgs/openimagedenoise/template]: done

File: srcpkgs/openimagedenoise/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'openimagedenoise'
+pkgname=openimagedenoise
+version=1.1.0
+revision=1
+archs="x86_64*"
+wrksrc=oidn-${version}
+build_style=cmake
+hostmakedepends="python3"
+makedepends="tbb-devel"
+depends="python3"
```
Comment (xtraeme) [inline srcpkgs/openimagedenoise/template]: I think it's not necessary at run-time?

Comment (xtraeme) [reply srcpkgs/openimagedenoise/template]: right, it's only required at build time. You can remove this `depends` line.

Comment (teldra) [reply srcpkgs/openimagedenoise/template]: done

**Pull request merged**